### PR TITLE
feat(blog): add a post about Resoto CDK

### PR DESCRIPTION
**Pull request by meln1k**
# Description

A new post about resoto-cdk

# To-Dos

- [x] Format with `yarn format`
- [x] Lint with `yarn lint`
- [x] Build successfully with `yarn build`

# Issues Fixed

- Fixes #XXXX

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: blog/2022/10-20-resoto-cdk/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+authors: [nikita]
+tags: [aws, cdk, installation]
+---
+
+# Resoto AWS CDK construct
+
+We recently released a [AWS CDK construct](https://eu-central-1.console.aws.amazon.com/cloudformation/home?region=eu-central-1#/stacks/create/review?templateURL=https://resoto-cdk.s3.eu-central-1.amazonaws.com/Resoto_EKS.template&stackName=ResotoEKS), which simplifies the deployment of Resoto to AWS.
```
Comment (TheCatLady) [inline blog/2022/10-20-resoto-cdk/index.md:8]: Hmm, this link is not accessible when you are not logged in. Is there another link we can use here?

File: blog/2022/10-20-resoto-cdk/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Getting started
+
```
Comment (TheCatLady) [inline blog/2022/10-20-resoto-cdk/index.md:11]:
```suggestion
```

File: blog/2022/10-20-resoto-cdk/index.md (hunk 1 above)
Comment (meln1k) [reply blog/2022/10-20-resoto-cdk/index.md:8]: I'm afraid not.

Comment (meln1k) [reply blog/2022/10-20-resoto-cdk/index.md:8]: good catch. I will update it

**Review by TheCatLady — APPROVED** [review]

**Pull request merged**